tests/core/creators/_helpers/tools.js

```diff
@@ -42,7 +44,7 @@ var detachedTests = ( function() {
 					delayIfDetached: true
 				} );
 
-				assert.isNull( editor, 'Editor should not be created on detached element, if config allows a delay. Creator function used: ' + creatorFunction );
+				assert.areSame( 'function', typeof editor, 'Editor should return function that allows to cancel creation. ' + assertMessage );
```

Comment:
This test is not good enough. It only checks the return type of the creation method, but we are not testing the outcome of using that callback.